src/mail/mail.module.ts

```
@@ -0,0 +1,9 @@
+import { Module } from '@nestjs/common';
+import { TypeOrmModule } from '@nestjs/typeorm';
```

ESLint

[error] 'TypeOrmModule' is defined but never used.

(@typescript-eslint/no-unused-vars)